api/pkg/api/config_group.go

```diff
@@ -46,9 +46,18 @@ func getPayload[T interface{}](c *fiber.Ctx) T {
 	return c.Context().UserValue("payload").(T)
 }
 
+// TODO: rename to parseRequestBody as part of https://czi-tech.atlassian.net/browse/CCIE-1005
 func parsePayload[T interface{}](c *fiber.Ctx) error {
+	return parsePayloadImpl[T](c, c.BodyParser)
+}
+
+func parseQueryString[T interface{}](c *fiber.Ctx) error {
+	return parsePayloadImpl[T](c, c.QueryParser)
+}
+
+func parsePayloadImpl[T interface{}](c *fiber.Ctx, fn request.RequestParser) error {
```

Comment:
I'm not a super fan of the naming convention of "impl" suffixes. Doesn't really describe what its doing

Reply:
trying to find a go reference for this comment one moment

Reply:
Fair, this is temporary though, I have another ticket to come make some more changes to this area and wanted to keep it separate since this is already a large PR